src/Adapters/CommonShared/SdkVersionUtils.cs

```diff
@@ -0,0 +1,27 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="SdkVersionUtils.cs" company="Microsoft">
+// Copyright (c) Microsoft Corporation. 
+// All rights reserved.  2013
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Microsoft.ApplicationInsights.Implementation
+{
+    using System;
+    using System.Linq;
+    using System.Reflection;
+
+    internal class SdkVersionUtils
+    {
+        internal static string GetSdkVersion(string versionPrefix)
+        {
+            string versionStr = typeof(SdkVersionUtils).Assembly.GetCustomAttributes(false)
+                    .OfType<AssemblyFileVersionAttribute>()
+                    .First()
+                    .Version;
+
+            Version version = new Version(versionStr);
+            return (versionPrefix ?? string.Empty) + version.ToString(3) + "-" + version.Revision;
```

Comment:
One is product code, second one is test code. I want them to be different so when you change product code and introduce a bug you catch it.


---
In reply to: [66372527](https://github.com/Microsoft/ApplicationInsights-dotnet-logging/pull/34#discussion_r66372527) [](ancestors = 66372527)